docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "System Table Configuration"
+date: 2024-01-23T16:30:21+05:30
+---
+
+## Configuration changes required for System Table Integration
+
+There is no unique configuration process for System Table ; however, the format for the **BELOW CONFIGURATIONS**
+are specific for System Table Integration. For all other configurations, please follow the
+[Configuration]({{%relref "DeployOverwatch/ConfigureOverwatch/Configuration.md"%}})
+
+* **auditlogprefix_source_path** - Instead of adding a fully qualified path (s3 or GC) for auditlog, 
+add keyword **system** in this column. This will enable the system table integration.
+
+**Same congifuration changes can be used for migrating from legacy to system table integration.**
```

Comment:
If customers are upgrading to 0800 they can't use legacy.

Reply:
done